Content.Server/DeltaV/StationEvents/Components/GlimmerMobRuleComponent.cs

```diff
@@ -0,0 +1,11 @@
+using Content.Server.StationEvents.Events;
+using Robust.Shared.Prototypes;
+
+namespace Content.Server.StationEvents.Components;
+
+[RegisterComponent, Access(typeof(GlimmerMobRule))]
+public sealed partial class GlimmerMobRuleComponent : Component
+{
+    [DataField(required: true)]
```

Comment:
```suggestion
    [DataField("mobPrototype")]
```

Reply:
for a while tags in datafields arent required, it defaults to the field name with first letter lowercased which is almost always what you want. required: true is needed since otherwise it lets you add the component without actually spawning anything

Reply:
Not many things use the required field anyways, and I dont see how this'd ever get in without something set anyways

Reply:
yes which is why the field is required, so if you are making a new rule it errors out making it easier to catch a mistake

a lot of modern code uses required when things are required, if you are looking at legacy stuff of course you wont see it that often